_maps/skyrat/automapper/automapper_config.toml

```diff
@@ -459,3 +459,102 @@ directory = "_maps/bubber/automapper/templates/generic/"
 required_map = "CentCom_skyrat_z2.dmm"
 coordinates = [72, 183, 2]
 trait_name = "CentCom"
+
+
+## Book of kindred display cases
+
+#DeltaStation2
+[templates.deltastation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "DeltaStation2.dmm"
+coordinates = [186, 105, 1]
+trait_name = "Station"
+
+#IceBoxStation
+[templates.iceboxstation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "IceBoxStation.dmm"
+coordinates = [175, 140, 3]
+trait_name = "Station"
+
+#LimaStation
+[templates.limastation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "LimaStation.dmm"
+coordinates = [99, 128, 2]
+trait_name = "Station"
```

Comment:
Depreciated map we also own. Not needed to automap it, but we also don't maintain it anymore